app/models/date_range.rb

```diff
@@ -0,0 +1,49 @@
+# == Schema Information
+#
+# Table name: date_ranges
+#
+#  id               :integer          not null, primary key
+#  event_id         :integer          index
+#  start_date       :date
+#  end_date         :date
+#  created_at       :datetime         not null
+#  updated_at       :datetime         not null
+
+#
+class DateRange < ActiveRecord::Base
+
+
+  belongs_to :event
+
+  validate :validate_start_date_not_in_past
+  validate :validate_end_not_before_start
+
+  def validate_start_date_not_in_past
+    if start_date < Date.today
+      errors.add(:start_date, "darf nicht in der Vergangenheit liegen.")
```

Comment:
I think since i18n wasn't mentioned as a goal for this sprint we can leave it out for now. But somehow everyone from other teams is going crazy to implement it in different ways. I don't want to decide this, but from my point of view german should be okay for this PR.